games/b.yaml

```
@@ -991,6 +991,27 @@
   video:
     youtube: SgVdsrDAUwA
 
+- name: Bomberman LAN
+  originals:
+  - Bomberman
+  type: clone
+  url: 'https://oldschoolprg.x10.mx/projets.php#:~:text=Bomberman%20LAN'
+  development: complete
+  status: playable
+  multiplayer:
+  - LAN
+  content: free
```

```suggestion
  content: commercial
```
it looks like it's using ripped assets